### ISSUE TITLE
Pyratite- A great concept worth saving

### ISSUE THREAD

**Issue by DS-3K** [Anuken/Mindustry-Suggestions#80]
**Describe what you would like changed, and why.**

I suggest improving the usefullness for pyratite, as it is rarely worth the time and sand/coal spend setting it up.

**Describe the changes you want to propose. Include possible alternatives.**
Introduction:
Pyratite is a very unique material - its factory is the only one to take three materials and it is one of few that ca not be sent to the core. Now the usage for pyratite is for ammunition and energy production, the problem here is that getting pyratite is often not worth it, and here is why.

Pyratite as ammunition:
Early game weapons:
+Very powerfull and usefull for most early game weaponry
-Producing pyratite is too time costly and it is better to upgrade to mid game weapons with your time instead
Mid game weapons:
+Still usefull for most weapons, but less choices than with early game weapons
- Is outclassed by lancer and thorium ammunition for the salvo
Late game weapons
+ Here there are also many choices to use pyratite ammunition
- Pyratite is outclassed by thorium ammunition or blast ammunition, it´s technically a early game material (copper and lead build cost, low energy maintainance) and can´t keep up with other choices.
Suggestion: Making pyratite the strongest choice for ammunition and balancing with the ammo multiplier is the only way I can see it beeing a considerable option for ammunition. Pyratite is something you have to make just so you can use it for ammo/energy, I think it is logical that it should be better than materials you have for building purposes, especially because pyratite is not fast to get and coal/sand are precious for many other materials.

Pyratite as a energy source:
Early game:
+Pyratite fuel gives the most energy
-Pyratite production takes too much energy for a quick net positive and it takes to long to set it up anyway
Mid game:
+Pyratite fuel gives way more power than other options
- it is outclassed by using coal quantatively
- Differantial generators are outclassed by thorium generators, as thorium is easier to get than pyratite in mid game
Late game
+ Can be used to make blast compount
- Pyratite has pretty much no use in late game on its own, as every other option is more provitable

Suggestion: Buffing pyratites power gain or making the differantial generator low maintainance is the only way I can see beiing able to consider getting pyratite in early or mid game. Right now, it is merely a substitute for the campaign before you can mine thorium and pyratite deserves to be usefull outside of early campaign levels.

Closing statement:
Pyratite is a unique and wonderfull material, one of my favourites even, and that´s why I´m asking for you to save it. Mindustry is a wonderfull, well balanced game and it hurts my heart to see something not work out as intended. I request fixing pyratite so it can be the awesome incedary ammunition and powerfull energy source it was meant to be (I think it was meant to be that at least)
Make Pyratite great again!

__Suggestion from [Steam] Dark_slayer3000__

**Comment by DS-3K** [comment]
I hope you like my Thesis xD

**Comment by CNC-er** [comment]
From my experience (having completed the campaign and multiple custom maps) the only turret where pyratite is a viable ammunition is the scorch. Pyratite loaded, cryo cooled phase overdriven scorches melt even through eradicators as if they are butter, there is nothing that can compare to the sheer dps of properly boosted scorches, it is the best ammo for ground defense if you know that there will be many eradicators knocking on your doorstep. From my experiences nothing holds a candle to fuses for air defense which require a relatively vast array of industry in order to be able to even place down fuse.

I would also like to say that pyratite is quite easy to acquire on most maps compared to a material like plastanium which requires the production of graphite, titanium, spore pods, alot of power, and alot of water for a usually a trickle of plastanium. Compared to just coal, sand, and lead which can all be mined with the lowest level drills and pyratite production is not energy intensive compared to plastanium production. So pyrite being an inferior ammo makes much sense.

Pyratite versus thorium also makes alot of sense in the current state. You need Laser drills to get thorium which are energy intensive and require the production of silicon, graphite and titanium. So once again its far easier to obtain pyratite than an alternative. 

And in terms of power, nothing beats Impact reactors which while being one of the most expensive buildings in the game to build are necessary in the late late game when thousands of unit assault your walls and sea's of mend projector and phase conveyors are necessary to keeps walls from being deleted and turrets fed with ammo. And all the resource production to allow the building of everything also consumes massive amounts of power. You need blast compound to fuel impact reactors which requires pyratite production to create blast compound so pyratite in the late late game is extremely important to make survival possible. 

So basically from my point of view: Pyratite is easy to aquire compared to other materials which are better for ammo. For power pyratite is essential in the late late game. The current value of pyratite compared to other materials is very well balanced.
